WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -158,7 +158,7 @@ extension PluginDirectoryEntry: Codable {
         guard let name = responseObject["name"] as? String,
             let slug = responseObject["slug"] as? String,
             let authorString = responseObject["author"] as? String,
-            let rating = responseObject["rating"] as? Double else {
+            let rating = responseObject["rating"] as? Int else {
```

Comment:
This reverts Emily's [fix](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/370), to show the test would have caught the failure in decoding the API response.